src/ui/widgets/rjsf/widgets/data-binding.tsx

```diff
@@ -44,8 +44,8 @@ import { getBlockComponent, getChaiDataProviders } from "@chaibuilder/runtime";
 import { ErrorBoundary } from "../../../../core/components/ErrorBoundary.tsx";
 import { useSelectedBlock } from "../../../../core/hooks";
 import { useChaiExternalData } from "../../../../core/components/canvas/static/useChaiExternalData.ts";
-
-const LazyJsonViewer = React.lazy(() => import("react-json-view"));
+import { JsonView, allExpanded, defaultStyles } from "react-json-view-lite";
```

Comment:
Peform a lazy import instead of direct import. It will increase the bundle size.